drush/README.md

```diff
@@ -0,0 +1,3 @@
+The three subdirectories here hold project-specific commandfiles, site aliases, and configuration for Drush. These will all be included automatically when you use the ./dr wrapper script that should be in your project root directory. The dr script intentionally skips loading commands/aliases/config from the typical global locations in order to ensure that all team members have same Drush environment.
+ 
+ See http://packages.drush.org/ for a directory of Drush commands installable via Composer.
```

Comment:
AFAIK, packages.drush.org is just query tool for packagist.org. So i'm thinking it does not need to be listed in 'repositories'.